Tasks/Maven/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -55,9 +55,11 @@
   "loc.input.help.sqDbPassword": "SonarQube server 5.1 and lower only. Enter the password for the database user i.e. sonar.jdbc.password",
   "loc.input.label.sqAnalysisIncludeFullReport": "Include full analysis report in the build summary (SQ 5.3+)",
   "loc.input.help.sqAnalysisIncludeFullReport": "This option will delay the build until the SonarQube analysis is completed.",
+  "loc.input.label.sqAnalysisBreakBuildIfQualityGateFailed": "Fail the build on quality gate failure (SQ 5.3+)",
```

Comment:
Unrelated change - the resjson has uncommited messages. This will lead to loc problems.